ratatool-sampling/src/main/scala/com/spotify/ratatool/samplers/BigSampler.scala

```diff
@@ -151,6 +167,15 @@ object BigSampler extends Command {
       log.warn("No seed specified, won't guarantee cohorts between datasets.")
     }
 
+    if (distribution.isEmpty) {
+      log.warn("No distribution specified, won't guarantee output distribution")
+    }
+
+    if (distribution.isDefined && distributionFields.isEmpty) {
```

Comment:
what about the inverse when you specify `distributionFields` but not a `distribution` now